src/main/java/model/Entities/Rental.java

```diff
@@ -1,4 +1,25 @@
 package model.Entities;
 
 public class Rental {
+    //primary id of Rental
+    int rid;
+
+    //identifies the Vehicle involved in the Rental
+    int vlicense;
+
+    //identifies the Customer involved in the Rental
+    long dlicense;
+
+    //time period over which Rental occurs
+    TimePeriod timePeriod;
+
+    int odometer;
+
+    //Card for which rental was made
```

Comment:
do we need something to store if it is MasterCard or Visa?

Reply:
I think cardName stores info like MasterCard/Visa

Reply:
Oh I thought it was the "name on the card" i.e. customer name. I'm thinking too much in terms of real world scenario :P